test/unit/has_slug_tests.rb

```diff
@@ -38,12 +38,36 @@ def update_column(name, value)
 
       value  = Factory.string
       record = subject.new.tap{ |r| r.send("#{@source_attribute}=", value) }
-      assert_instance_of Proc, subject.ardb_has_slug_config[:source_proc]
-      assert_equal value, record.instance_eval(&subject.ardb_has_slug_config[:source_proc])
+      source_proc = subject.ardb_has_slug_config[:source_proc]
+      assert_instance_of Proc, source_proc
+      assert_equal value, record.instance_eval(&source_proc)
 
       upcase_value = value.upcase
-      assert_instance_of Proc, subject.ardb_has_slug_config[:preprocessor_proc]
-      assert_equal value, subject.ardb_has_slug_config[:preprocessor_proc].call(upcase_value)
+      preprocessor_proc = subject.ardb_has_slug_config[:preprocessor_proc]
+      assert_instance_of Proc, preprocessor_proc
+      assert_equal value.downcase, preprocessor_proc.call(upcase_value)
```

Comment:
@kellyredding - I wanted to explicitly downcase the value (not really on `Factory.string` always returning a downcased string), but then that pushed it way over 80 characters so I saved the proc into a local var. That's why all these lines changed.